package.json

```diff
@@ -26,14 +26,12 @@
     "eslint-config-airbnb-base": "^15.0.0"
   },
   "peerDependencies": {
-    "@typescript-eslint/eslint-plugin": "^5.13.0",
-    "@typescript-eslint/parser": "^5.0.0",
+    "@typescript-eslint/eslint-plugin": "^5.13.0 || ^6.0.0",
+    "@typescript-eslint/parser": "^5.0.0 || ^6.0.0",
     "eslint": "^7.32.0 || ^8.2.0",
     "eslint-plugin-import": "^2.25.3"
   },
   "devDependencies": {
-    "@typescript-eslint/eslint-plugin": "5.15.0",
```

Comment:
Those dependencies must not be removed from devDependencies because they are used on _lib/shared.js_, which is imported on base.js and extended on .eslintrc.js

Reply:
This is a bit strange (since there are no "ts" files in the project) but true.
In addition, everything works now - the peer dependencies, as I checked, are also installed, so I did not see any errors after the removal.
But for clarity, I returned the records. Thank you.